troubleshooting/known_issues/admin_client_auth_not_found.md

```diff
@@ -0,0 +1,40 @@
+# Known Issue: `admin-client-auth` Not Found
+
+Running the Install CSM Services script, the following error may occur:
+
+```text
+ERROR   Step: Set Management NCNs to use Unbound --- Checking Precondition
++ Getting admin-client-auth secret
+Error from server (NotFound): secrets "admin-client-auth" not found
++ Obtaining access token
+```
+
+## Fix
+
+This can occur if the `keycloak-users-localize` pod has not completed, and that can be caused by an intermittent `Istio` issue. Remediate the issue with the following procedure:
+
+   1. Follow [Troubleshoot Intermittent HTTP 503 Code Failures](../../operations/kubernetes/Troubleshoot_Intermittent_503s.md) to verify that `Istio` is healthy.
+
+   1. Ensure that the `keycloak-wait-for-postgres-*` pod is in a `Completed` state.
+
+   Example output:
+
+   ```text
+   kubectl get po -n services | grep keycloak-wait-for-postgres
+   keycloak-wait-for-postgres-1-pv85m                                0/2     Completed   0          15d
+   ```
+
+   1. If the `keycloak-wait-for-postgres-*` pod is not in a `Completed` state, resubmit the job:
+
+   ```bash
+   kubectl get job -n services keycloak-wait-for-postgres-1 -o json | jq 'del(.spec.selector)' | jq 'del(.spec.template.metadata.labels)' | kubectl replace --force -f -
```

Comment:
I've seen systems where a newer job was created and the current one is is not keycloak-wait-for-postgres-1, might what to make this more generic with:
kubectl get job -n services -l app.kubernetes.io/instance=cray-keycloak-users-localize -o json | jq 'del(.spec.selector)' | jq 'del(.spec.template.metadata.labels)' | kubectl replace --force -f -

Reply:
i like it -- will fix

Reply:
fixed